src/pages/Upstream/components/Step1.tsx

```diff
@@ -43,9 +47,38 @@ const initialValues = {
     send: 6000,
     read: 6000,
   },
+  active: false,
+  passive: false,
+  checks: {
+    active: {
+      timeout: 5,
+      http_path: '',
+      host: '',
+      healthy: {
+        interval: 2,
+        successes: 1,
+      },
+      unhealthy: {
+        interval: 1,
+        http_failures: 2,
+      },
+      req_headers: [''],
+    },
+    passive: {
+      healthy: {
+        http_statuses: [undefined],
```

Comment:
could we use an empty array here?

Reply:
empty array will lose  the first item  while rendering Form.List 
we tried to use [''], but the type of the http_statuses: number[], so we use [undefined] here

Reply:
got it, thanks!